revscoring/features/wikibase/datasources/revision_oriented.py

```diff
@@ -114,7 +115,10 @@ def __init__(self, name, revision_datasources):
 
 def _process_entity_doc(text):
     if text is not None:
-        return json.loads(text)
+        try:
+            return json.loads(text)
+        except json.decoder.JSONDecodeError:
+            raise UnexpectedContentType(text)
```

Comment:
To be clear, I recommend saying something specific about "Entity JSON" here.

OR raising an error that is more specific to Entity JSON parsing that extends UnexpectedContentType

Given that I think this is the only place we might throw such an error, I'm more in favor of the former, but I could see value in the latter.  It's nice to have the consistent parts of error text defined in errors.py

Reply:
Definitely, I added a specific parameter to the Exception, lemme know what you think about it.